src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ArmConstants;
+
+public class ArmSubsystem extends SubsystemBase {
+
+  WPI_TalonSRX rotateArmMotor;
+  WPI_TalonSRX grabArmMotor;
+
+  DigitalInput limitSwitchUp;
+  DigitalInput limitSwitchDown;
+
+  public ArmSubsystem() {
+    rotateArmMotor = new WPI_TalonSRX(ArmConstants.kRotateArmMotorPort);
+    grabArmMotor = new WPI_TalonSRX(ArmConstants.kGrabArmMotorPort);
+
+    limitSwitchUp = new DigitalInput(ArmConstants.kLimitSwitchUpPort);
+    limitSwitchDown = new DigitalInput(ArmConstants.kLimitSwitchDownPort);
+  }
+
+  public void rotateArm(double value){
+    if (limitSwitchUp.get()) {
+      rotateArmMotor.set(TalonSRXControlMode.PercentOutput, 0);
+    } else if (limitSwitchDown.get()) {
+      rotateArmMotor.set(TalonSRXControlMode.PercentOutput, 0);
+    } else {
+      rotateArmMotor.set(TalonSRXControlMode.PercentOutput, value);
+    }
+  }
+
+  public void grabArm(double value){
+    grabArmMotor.set(TalonSRXControlMode.PercentOutput, value);
```

Comment:
Isn't the control mode `PercentageOutput` by default?

Reply:
right, I'll change.